qemu/tests/cfg/smbios_default_check.cfg

```diff
@@ -0,0 +1,24 @@
+- smbios_default_check: install setup image_copy unattended_install.cdrom
+    only q35
+    no Host_RHEL.m6,Host_RHEL.m7,Host_RHEL.m8.u0,Host_RHEL.m8.u1
+    # dmidecode not available (and no plans for it) on s390
+    no s390x
+    type = smbios_default_check
+    requires_root = yes
+    start_vm = yes
+    kill_vm = yes
+    # Expected output
+    System_Manufacturer = "^Red Hat$"
+    System_SKU_Number = "^[89].[23456789].0$"
+    Baseboard_Manufacturer = "^Red Hat$"
+    Baseboard_Product_Name = "^RHEL-AV$"
```

Comment:
The above definition might not apply to fedora, right? So maybe restrict it to `RHEL:`?

Reply:
And, only `RHEL-AV`? So it might need to specify qemu version?

Reply:
Yes, this only applies to rhel, I will limit it to RHEL later.
"**Baseboard_Product_Name**" value can only be "**RHEL-AV**", and qemu should be higher than 4.2.0,and I want add "required_qemu = [4.2.0,)",do you agree with me? Thanks!

Reply:
> Yes, this only applies to rhel, I will limit it to RHEL later.
> "**Baseboard_Product_Name**" value can only be "**RHEL-AV**", and qemu should be higher than 4.2.0,and I want add "required_qemu = [4.2.0,)",do you agree with me? Thanks!

Yes, please.